### PR TITLE
DEV-1483: Implement IDPartner-hosted Trust Platform

### PR DESCRIPTION
**Pull request by francisco-idpartner**
Implement IDPartner-hosted Trust Platform.

Update the repo to have two subfolders: the new idpartner-hosted and the pre-existing self-hosted trust platform.

Instructions were added to manually get the trust-platform up and running. However, as this project evolves it makes more sense that the trust-platform-example can be downloaded from the IDPartner console. In fact, it's already possible. The IDPartner console uses this repo to generate a ready-to-use trust platform example.

https://identity-online.atlassian.net/browse/DEV-1441
https://identity-online.atlassian.net/browse/DEV-1483
https://identity-online.atlassian.net/browse/DEV-1471

### CODE REVIEW

File: idpartner-hosted/docs/prerequisites.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Prerequisites
+Before proceeding, ensure the following tools are installed on your system as they are essential for completing the setup.
+
+| Tool    | Instructions                                                           | Verification Command | Supported Version  |
+| ------- | ---------------------------------------------------------------------- | -------------------- | ------------------ |
+| ngrok   | [Download from ngrok](https://ngrok.com/download)                      | `ngrok version`      | 3.1.0 and above    |
+| nodejs  | [Download from Node.js](https://nodejs.org/en/download/)               | `node -v`            | v16.16.0 and above |
```
Comment (yovasx2) [inline idpartner-hosted/docs/prerequisites.md:7]: i think minimum is 18 no?

File: idpartner-hosted/docs/prerequisites.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+| yarn    | [Download from Yarn](https://classic.yarnpkg.com/lang/en/docs/install) | `yarn --version`     | 1.18.0 and above   |
+
+Install dependencies. Run `yarn install` and make sure it completes without errors.
+
+## Install Dependencies
+After installing the tools, run the following command to install the necessary dependencies. Ensure that this process completes without any errors.
+```bash
+yarn install
```
Comment (yovasx2) [inline idpartner-hosted/docs/prerequisites.md:15]: Rod made me change to npm, in template download, won't it be the same?

File: idpartner-hosted/docs/prerequisites.md (hunk 1 above)
Comment (francisco-idpartner) [reply idpartner-hosted/docs/prerequisites.md:7]: It works with 16.x.x too. No problem

File: idpartner-hosted/docs/prerequisites.md (hunk 2 above)
Comment (francisco-idpartner) [reply idpartner-hosted/docs/prerequisites.md]: Updated to use npm

**Pull request merged**